app/src/main/java/com/yubico/yubikitold/YubiKitManager.java

```diff
@@ -20,8 +20,8 @@
  */
 public final class YubiKitManager {
     private final Handler handler;
-    private final UsbDeviceManager usbDeviceManager;
-    private final NfcDeviceManager nfcDeviceManager;
+    private final com.yubico.yubikitold.transport.usb.UsbDeviceManager usbDeviceManager;
```

Comment:
do you need this prefix when you've got import?

Reply:
Not really! Android Studio's refactoring included that and I ended up not noticing it. Thanks for pointing it out.